openedx/core/djangoapps/cors_csrf/tests/test_authentication.py

```diff
@@ -7,16 +7,21 @@
 from django.test.client import RequestFactory
 from django.conf import settings
 
+from rest_framework.authentication import SessionAuthentication
 from rest_framework.exceptions import PermissionDenied
 
-from ..authentication import SessionAuthenticationCrossDomainCsrf
+from ..authentication import SessionAuthenticationCorsCSRFMixin
 
 
 # A class to pass into django.middleware.csrf.get_token() so we can easily get a valid CSRF token to use.
 class FakeRequest(object):
     META = {}
 
 
+class CrossDomainSessionAuthentication(SessionAuthenticationCorsCSRFMixin, SessionAuthentication):
```

Comment:
Seems odd that this was changed to a mixin with its only use in this test?  Should this just be deleted?  Otherwise, it would be great to see it used elsewhere to get why it is better this way. :)